src/test/OptionSettlement.t.sol

```diff
@@ -91,56 +95,62 @@ contract OptionSettlementTest is DSTest, NFTreceiver {
             settlementSeed: 1234567,
             underlyingAmount: 1 ether,
             exerciseAmount: 3000 ether,
-            exerciseTimestamp: uint40(block.timestamp),
+            exerciseTimestamp: uint40(block.timestamp) + 100,
```

Comment:
You'll need to increment the line below this as well @cartercarlson or you'll get failures.